ui/src/main/webapp/src/app/components/reports/migration-issues/migration-issues-table.component.ts

```diff
@@ -118,4 +118,33 @@ export class MigrationIssuesTableComponent implements OnInit {
 
         this.sortedIssues = this._sortingService.sort(this.migrationIssues);
     }
+
+    showRule(ruleID: string) {
+        /*
+        This is not working either.
```

Comment:
Interesting. Maybe there is some issue with '' parent routes? I don't know...
I tried both `../ `and `../../`   but I always got route 
`http://localhost:8080/windup-web/projects/9238/groups/9239/reports/95770/migration-issues/executed-rules?ruleID=os-specific-00001`